pkgs/top-level/all-packages.nix

```diff
@@ -30882,7 +30882,9 @@ with pkgs;
   inherit (callPackage ../desktops/gnome/extensions { })
     gnomeExtensions
     gnome38Extensions
-    gnome40Extensions;
+    gnome40Extensions
+    gnome41Extensions
```

Comment:
Is there a way to simply add all attributes of the file to `all-packages.nix`? We already forgot that line in the GNOME 41 update, so it is quite probable that things will be forgotten again. The only solution I can think of would be to remove this and add `// (callPackage ../desktops/gnome/extensions {})` at the end. But somehow am not that fond of that idea.

Reply:
Only alternative I can think of would be gnome.extensions41 and you probably don't like that either.

Reply:
You could add a line to the [README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/desktops/gnome/extensions/README.md) which should decrease the chance of it being forgotten.

Reply:
Done.